src/api_routes_http.rs

```diff
@@ -277,8 +267,12 @@ pub fn config(cfg: &mut ServiceConfig, rate_limit: &RateLimitCell) {
           .route("/save", put().to(save_comment))
           .route("/list", get().to(list_comments))
           .route("/report", post().to(create_comment_report))
-          .route("/report/resolve", put().to(resolve_comment_report))
-          .route("/report/list", get().to(list_comment_reports)),
+          .route("/report/resolve", put().to(resolve_comment_report)),
```

Comment:
How about removing these separate report endpoints and making a single endpoint `/report/resolve` which takes `ReportCombinedId`?

Reply:
Probably a not a good idea, since you still need to create report for the specific item anyway. And the `reportcombinedid` isn't useful for resolving.